core/application/impl/app_config_impl.cpp

```diff
@@ -87,7 +87,7 @@ namespace kagome::application {
   }
 
   void AppConfigurationImpl::parse_general_segment(rapidjson::Value &val) {
-    uint16_t v;
+    uint16_t v; // NOLINT
```

Comment:
```suggestion
    uint16_t v{};
```
Why not initialize it?

Reply:
```load_u16(val, "verbosity", v)``` is the initialization. It is not necessary to init twice.